assets/package.json

```
@@ -3,7 +3,8 @@
   "scripts": {
     "deploy": "webpack --mode production",
     "stylelint": "stylelint 'css/**/*.scss'",
-    "watch": "webpack --mode development --watch-stdin"
+    "watch": "webpack --mode development --watch-stdin",
+    "lint": "./node_modules/.bin/eslint ."
```

Thoughts on…

```suggestion
    "eslint": "./node_modules/.bin/eslint ."
```

…since we have other linting, too?